ResearchKit/Consent/ORKVisualConsentStepViewController.m

```diff
@@ -391,12 +415,23 @@ - (void)finishTransitioningAnimator:(ORKVisualConsentTransitionAnimator *)animat
     }
 }
 
+- (void)observedScrollViewDidScroll:(UIScrollView *)scrollView {
+    CGRect animationViewFrame = _animationView.frame;
+    CGPoint scrollViewBoundsOrigin = scrollView.bounds.origin;
+    animationViewFrame.origin = (CGPoint){-scrollViewBoundsOrigin.x, -scrollViewBoundsOrigin.y};
+    _animationView.frame = animationViewFrame;
+}
+
 - (void)showViewController:(ORKConsentSceneViewController *)viewController forward:(BOOL)forward animated:(BOOL)animated {
 
     if (! viewController) {
         return;
     }
     
+    // Stop old observer and start new one
+    _scrollViewObserver = [[ORKScrollViewObserver alloc] initWithTargetView:viewController.scrollView responder:self];
```

Comment:
I think I disagree about clearing the old observer's `delegate`, it feels as needless visual code pollution.

I made extra sure the delegate is deallocating on `_scrollViewObserver` reassign (which happens after the next button is pressed and before the next *viewController* is shown). It's the same approach as in `ORKTaskViewController`'s `setRegisteredScrollView:`.

![screen shot 2015-04-23 at 10 49 51 2](https://cloud.githubusercontent.com/assets/444313/7294584/8b03265c-e9a6-11e4-8c30-8ecf80b671f5.png)

If you disagree, let me know and I shall add it anyway.